app/templates/categories/routing-map.html

```diff
@@ -0,0 +1,100 @@
+{% extends "base.html" %}
```

Comment:
Should we have robot/nofollow tags here?

Reply:
Yes we should, however, I think we should do this as a general ticket rather than for this specific case.